tools/PicoLoaderConverter/PicoLoaderConverter/ApList/ApListFactory.cs

```diff
@@ -172,6 +172,7 @@ private DSProtectVersion ParseDSProtectVersion(string dsProtectVersion)
             "2.01s" => DSProtectVersion.V2_01s,
             "2.03s" => DSProtectVersion.V2_03s,
             "2.05s" => DSProtectVersion.V2_05s,
+            "1.00_2" => DSProtectVersion.V1_00_2,
```

Comment:
Although the enum is not sorted, you can keep this switch sorted.

Please add to `FormatDSProtectVersion` as well